src/sonic-host-services/scripts/determine-reboot-cause

```diff
@@ -98,18 +98,23 @@ def find_proc_cmdline_reboot_cause():
 
     return proc_cmdline_reboot_cause
 
+
 def get_reboot_cause_from_platform():
     # Until all platform vendors have provided sonic_platform packages,
     # if there is no sonic_platform package installed, we only provide
     # software-related reboot causes.
+    hardware_reboot_cause_major = str()
+    hardware_reboot_cause_minor = str()
```

Comment:
Why initialize to `str()`? Maybe initialize to `"N/A"?

Reply:
`N/A` is better description of what is being done here. Updated the code, and the new output from syslog.